definitions/artifacts/azure-storage-account-blob.json

```diff
@@ -37,6 +37,16 @@
             }
           }
         },
+        "authentication": {
+          "title": "Authentication configuration",
+          "type": "object",
+          "properties": {
+            "sas_token": {
```

Comment:
Let's make the `sas_token` a required property. Its easy to remove `required`, its very hard to add it. If there is an `authentication` block, we should require it have a field.